.github/workflows/Test.yml

```diff
@@ -65,11 +66,21 @@ jobs:
           path: clang-format-output.txt
   
   license-headers:
-      name: Check license headers
+      name: License headers
       runs-on: ubuntu-18.04
       steps:
       - uses: actions/checkout@v1
       - name: Check license headers
         run: |
           chmod +x cmake/check_license.sh
           cmake/check_license.sh
+
+  pr-base-check:
+      name: Base branch
+      runs-on: ubuntu-18.04
+      if: github.event_name == 'pull_request'
+      steps:
+      - name: Fail, if PR base is master
+        if: contains(github.base_ref, 'master') && !contains(github.head_ref, 'development')
```

Comment:
change dev to release, also i suggest to change step name so it clearly describes the step